orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/RollbackServerGroupStage.groovy

```diff
@@ -91,14 +101,17 @@ class RollbackServerGroupStage implements StageDefinitionBuilder {
         parentStage.execution, enableServerGroupStage.type, "enable", enableServerGroupContext, parentStage, SyntheticStageOwner.STAGE_AFTER
       )
 
+      stages << buildCaptureSourceServerGroupCapacityStage(parentStage, parentStage.mapTo(ResizeStrategy.Source))
+
       Map resizeServerGroupContext = new HashMap(parentStage.context) + [
-        action : ResizeStrategy.ResizeAction.scale_to_server_group.toString(),
-        source : {
+        action            : ResizeStrategy.ResizeAction.scale_to_server_group.toString(),
+        source            : {
           def source = parentStage.mapTo(ResizeStrategy.Source)
           source.serverGroupName = rollbackServerGroupName
           return source
         }.call(),
-        asgName: restoreServerGroupName
+        asgName           : restoreServerGroupName,
+        pinMinimumCapacity: true
```

Comment:
`pinMinimumCapacity` is new but this particular ResizeAction (`scale_to_server_group`) is only used during a rollback.